tests/drivers/counter/counter_basic_api/testcase.yaml

```diff
@@ -3,3 +3,4 @@ tests:
     tags: drivers
     depends_on: counter
     min_ram: 16
+    platform_exclude: nucleo_f302r8
```

Comment:
Why not add min_flash instead?

Reply:
Because there are boards with small amount of flash that work perfectly fine.